.github/workflows/lock-closed-issues.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Lock Issues'
```

Comment:
Why is this file showing up here again when it's already present on master? https://github.com/phpstan/phpstan-mockery/blob/master/.github/workflows/lock-closed-issues.yml Perhaps this branch isn't rebased?

Reply:
Weird, I've forked the repository this morning. 
Rebasing.